### ISSUE TITLE
A chapter about physics simulation

### ISSUE THREAD

**Issue by vyskocil** [d-ronnqvist/SCNBook-code#12]
I miss very much a chapter explaining the physics simulation ie all the SCNPhysics* classes

**Comment by d-ronnqvist** [comment]
Yes, that's something that I want to get to. Realistically it's looking like something after the first two updated.
